Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,6038 @@
+        page_size (int, optional): Number of ip-list per page.
+        limit (int, optional): The maximum number of records to retrieve.
+    Returns:
+        Tuple[dict,str]: Output and paginate_results  message for Command Results.
+    """
+    response = response if isinstance(response, list) else [response]
+    page = arg_to_number(args.get('page'))
+    page_size = arg_to_number(args.get('page_size'))
+    limit = arg_to_number(args.get('limit', LIMIT_SIZE))
+
+    output = response
+
+    if page and page_size:
+        if page_size < len(response):
+            first_item = page_size * (page - 1)
+            output = response[first_item:first_item + page_size]
+        else:
+            output = response[:page_size]
+        pagination_message = f'Showing page {page}. \n Current page size: {page_size}'
+    else:
+        output = response[:limit]
+        pagination_message = f'Showing {len(output)} rows out of {len(response)}.'
+
+    return output, pagination_message
+
+
+def find_dict_in_array(container: List[Dict[str, Any]], key: str, value: str) -> Optional[Dict[str, Any]]:
+    """Gets dictionary object in list of dictionaries.The search is by key that exist in each dictionary.
+
+    Args:
+        container (list): List of dictionaries.
+        key (str): Key to recognize the correct dictionary.
+        value (str]): The value for the key.
+
+    Returns:
+        Optional[Dict[str, Any]]: The dictionary / The dictionaries / None if there is no match.
+    """
+
+    for obj in container:
+        if obj.get(key) and str(obj[key]) == value:
+            return obj
+    return None
+
+
+def find_dicts_in_array(container: List[Dict[str, Any]], key: str, value: List[str]) -> Optional[List[Dict[str, Any]]]:
+    """Gets dictionaries object in list of dictionaries.The search is by key that exist in each dictionary.
+
+    Args:
+        container (list): List of dictionaries.
+        key (str): Key to recognize the correct dictionary.
+        value (List[str]): The values for the key.
+
+    Returns:
+        Optional[List[Dict[str, Any]]]: The dictionaries / None if there is no match.
+    """
+
+    return [obj for obj in container if obj[key] in value]
+
+
+def get_object_id(client: Client,
+                  create_response: Dict[str, Any],
+                  by_key: str,
+                  value: str,
+                  get_request: Callable,
+                  object_id: Optional[str] = None) -> str:
+    """Get object / sub object id. After create sub (member)
+        object we should get list of all members and get our id by some key.
+
+    Args:
+        client (Client): Fortiweb VM Client.
+        create_response (Dict[str, Any]): Response from create request.
+        by_key (str): Unique key to search the sub object.
+        value (str): The sub object value to the key.
+        get_request (Callable): Get request (for Fortiweb VM 1)
+        object_id (Optional[str]): Object ID (not sub object
```

Comment:
this  should be a class method. Please go over all methods that have an `if client.version == ...` and consider bringing them into the client. Let me know if you have questions or this would take too long.